### PR TITLE
Base: Add/update some application manpages (Part 1)

### PR DESCRIPTION
**Pull request by cubiclove**
Add application manpages for:
- 3D File Viewer
- CatDog
- Certificate Settings

This contributes towards #11311

Also
- Update `Applications.md` to reflect these new pages and add previously missing pages.
- Update `Assistant.md` with instructions for:
  - Running terminal commands. Reflect this in the sister section in `Tips-and-Tricks.md` 
  - Launching applications with arguments.

### CODE REVIEW

File: Base/usr/share/man/man1/Applications/3DFileViewer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Name
+
+![Icon](/res/icons/16x16/app-3d-file-viewer.png) 3D File Viewer
+
+[Open](file:///bin/3DFileViewer)
+
+## Synopsis
+
+```**sh
+$ 3DFileViewer
```
Comment (BenWiederhake) [inline Base/usr/share/man/man1/Applications/3DFileViewer.md:10]: At the danger of [repeating](https://github.com/SerenityOS/serenity/issues/11311#issuecomment-997474629) myself, I suggest that:
- Either these things should also be displayed when `3DFileViewer --help` is run, i.e. these pages can and should be auto-generated to be kept in sync
- Or alternatively, the man page should not exist if it doesn't contribute anything.

The "synopsis" feels a bit weird, documenting all zero options. I like the "features" lists though, so I'd argu in favor of displaying these things both in the man page *and* in the `foo --help` output.

Comment (cubiclove) [reply Base/usr/share/man/man1/Applications/3DFileViewer.md:10]: The application manpages being autogenerated from `--help` seems like a larger decision the community needs to make (so far outside the scope of this PR).

I agree the synopsis seems unnecessary, but I'm following the conventions of the other app manpages - if I don't, someone else will request it.

However, I would agree with [Sam's comment](https://github.com/SerenityOS/serenity/issues/11311#issuecomment-999494877). The manpages can provide more context and explanation, like a wikipedia page for each application. That's what I've tried to do with the CatDog manpage especially.

Comment (BenWiederhake) [reply Base/usr/share/man/man1/Applications/3DFileViewer.md:10]: The application manpages *already are* being autogenerated from `--help`, for a good handful of programs. See `Meta/export-argsparser-manpages.sh` and `Base/root/generate_manpages.sh`. All that would be necessary is to add it to the list in `generate_manpages.sh`.

Comment (cubiclove) [reply Base/usr/share/man/man1/Applications/3DFileViewer.md:10]: According to the discussion you linked, there doesn't seem to be agreement that the manpages and `--help` output should always be the same (though I can agree it makes sense when there isn't anything to expand).
I'm going to leave these pages as is for now, but if the community decides to follow your idea then I'll be happy to adapt them :^)

**Review by GMTA — APPROVED** [review]

**Pull request merged**